### PR TITLE
Added tests for POST /_remotestore/_restore.

### PR DESCRIPTION
**Pull request by dblock**
### Description

Figuring out how to manufacture data to restore from ...

Added tests for POST /_remotestore/_restore.

### Issues Resolved

Part of #663.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: tests/remote_store/remote_store/restore.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+prologues:
+  - path: /_bulk
+    method: POST
+    parameters:
+      refresh: true
+    request:
+      content_type: application/x-ndjson
+      payload:
+        - {create: {_index: books, _id: book1}}
+        - {author: Harper Lee, title: To Kill a Mockingbird, year: 60}
```
Comment (andrross) [inline tests/remote_store/remote_store/restore.yaml:13]: Wow, I didn't realize this book was nearly 2000 years old.

**Review by andrross — APPROVED** [review]

File: tests/remote_store/remote_store/restore.yaml (hunk 1 above)
Comment (dblock) [reply tests/remote_store/remote_store/restore.yaml:13]: It's some bad copy paste from other tests that try to do "math". I updated the years where I could in this PR.

Comment (Tokesh) [inline tests/remote_store/remote_store/restore.yaml:13]: Sorry to being annoying, please make it 1960 and i can approve this PR

**Review by Tokesh — CHANGES_REQUESTED** [review]
small changes

File: tests/remote_store/remote_store/restore.yaml (hunk 1 above)
Comment (dblock) [reply tests/remote_store/remote_store/restore.yaml]: done

**Review by Tokesh — APPROVED** [review]

**Pull request merged**